### ISSUE TITLE
Uncaught SyntaxError: Unexpected token ~ (product 41)

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-server#1130]
```
Highcharts.setOptions({
	lang: {
		months: ~['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'~],
		weekdays: ~['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'~]
	}
});
```
https://us.openfoodfacts.org/products

**Comment by hangy** [comment]
This should be a duplicate of #771. Maybe the module has not been updated from cpan, yet?

**Comment by stephanegigandet** [comment]
I have updated the module from CPAN but I'm still getting the pb, in fact the t/lang.t test is not passing on the prod server:

ok 369 - footer_legal_link link - lang: fr - textid: mentions-legales -- file /lang/fr/texts/mentions-legales.html does not exist
ok 370 - footer_terms_link link - lang: fr - textid: conditions-d-utilisation -- file /lang/fr/texts/conditions-d-utilisation.html does not exist
not ok 371
#   Failed test at ../t/lang.t line 68.
#          got: '~['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'~]'
#     expected: '['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December']'
1..371
# Looks like you failed 1 test of 371.
root@ns3362784:/home/off/lib#  perl -le 'eval "require Locale::Maketext::Lexicon::Getcontext" and print ${"Locale::Maketext::Lexicon::Getcontext::VERSION"}'
0.05

**Comment by hangy** [comment]
Fixed by #1382
